web-src/src/components/NavbarBottom.vue

```diff
@@ -287,6 +299,7 @@ export default {
     PlayerButtonNext,
     PlayerButtonPlayPause,
     PlayerButtonPrevious,
+    PlayerButtonLyrics,
```

Comment:
Can you please order the components alphabetically?
Thanks in advance.

Reply:
Done